### DISCUSSION TITLE
metadata_check_validate test not seeing latest files ?

### DISCUSSION THREAD

**Discussion by mcdalvi** [MetOffice/simulation-systems#116] [UM]
Hello,
I  have [branch](https://code.metoffice.gov.uk/trac/um/log/main/branches/dev/mohitdalvi/vn13.4_fix_ukca_plev_diags) that fixes STASHmaster entries for some diagnostics that were incorrectly added. Some of the fixed stash items are now freshly requested in a rose-stem app. 
However, on running through rose-stem the [_spice_script_metadata_check_validate_](http://fcm1/cylc-review/taskjobs/hadzm?&suite=vn13.4_fix_ukca_plev_diags&no_fuzzy_time=0&tasks=script_spice_metadata_check_validate) test is failing with option code, etc errors that have been fixed and I suspect that it is not looking at the STASHmaster file in the branch.
Hardwiring the _meta=_ path to its own rose-meta/HEAD in a [test branch](https://code.metoffice.gov.uk/trac/um/changeset/121649/) allows [this test](http://fcm1/cylc-review/taskjobs/hadzm/?suite=plev_diags_script_check) to pass.

Are there any additional command-line options required for rose-stem when testing with changed metadata?

TIA

**Comment by james-bruten-mo** [comment]
Hi Mohit,

What you need here is a blank upgrade macro, eg. vn133_t7368 [here](https://code.metoffice.gov.uk/trac/um/browser/meta/trunk/um-atmos/version133_134.py#L53). That will change the path as you did manually, but the new version specified won't exist so it will look at HEAD which will be your branch in this case. Hopefully that makes sense.

Cheers,
James

**Reply by mcdalvi** [reply]
Thanks James,
That has worked in the test branch (after applying the macro to all apps!).
